grpc.go

```diff
@@ -524,6 +525,76 @@ func (a *agentGRPC) rollbackFailingContainerCreation(ctr *container) {
 	}
 }
 
+func changeToBundlePath(spec *specs.Spec) (string, error) {
```

Comment:
This (and `writeSpecToFile()`) are OCI-specific functions so please can you move them to `protocols/grpc/utils.go`.